resources/js/components/WbsIssues.vue

```diff
@@ -79,45 +78,34 @@
     },
 
     methods: {
-      newSibling(ofIssueIndex) {
+      newNode(ofIssueIndex, parentId, indentation = 0) {
```

Comment:
I think this method could be more readable if it has the following signature:

```suggestion
      newNode(parentId = null) {
```

With only the parent's ID, you can find the `ofIssueIndex` and `indentation`.

Reply:
According to this change, by removing the reference on the current node it is now impossible to check whether it is a new node or an existing node. This makes it possible to create several obsolete nodes that will be removed from the list during the next update cycle.